library/std/src/thread/local.rs

```diff
@@ -192,9 +192,11 @@ macro_rules! __thread_local_inner {
             // aren't implemented because thread dtors aren't really a thing
             // on wasm right now
             //
+            // uefi also maps directly to `static mut` since threads do not exist in uefi
+            //
             // FIXME(#84224) this should come after the `target_thread_local`
             // block.
-            #[cfg(all(target_family = "wasm", not(target_feature = "atomics")))]
+            #[cfg(any(all(target_family = "wasm", not(target_feature = "atomics")), target_os = "uefi"))]
```

Comment:
This not getting picked up by tidy is erroneous, and only is because `target_family` did not exist yet.

Reply:
Fixed by https://github.com/rust-lang/rust/pull/108917